string name
string map
string pointset
geometry_msgs/Pose pose
Vertex[] verts
Edge[] edges
